api/app/src/routes/webhook/index.ts

```diff
@@ -5,6 +5,7 @@ import { processAPIKey } from "../middlewares/auth";
 
 const routes = Router();
 
+// Need to think more about webhook for analytics
 routes.use("/garmin", garmin);
 routes.use("/apple", processAPIKey, apple);
```

Comment:
going to raise a follow-on issue for this, or?

Reply:
I added the work so i removed the comment